lib/BIGSdb/Application.pm

```diff
@@ -219,6 +219,24 @@ sub _initiate {
 	return;
 }
 
+sub _set_dbconnection_params {
```

Comment:
This needs to be named without the front underscore as it is called from outside of the module (by Offline::RunJobs), so it is not a private method.